docs/reference_guides/model_libraries/generic/unit_models/stream_scaler.rst

```diff
@@ -0,0 +1,46 @@
+Stream Scaler Block
+===================
+
+Stream Scaler Blocks are used to adjust size of streams to represent, for example, a stream being split across several identical units, which are then all modeled as a single IDAES unit
```

Comment:
An explanation of how this is done is important - some of this could be done by expanding the sections below.